exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/main_config.ipynb

```diff
@@ -7,9 +7,16 @@
    "source": [
```

Comment:
<p>that looks weird</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/113/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Oops</p>